sdk/logs/src/main/java/io/opentelemetry/sdk/logs/export/BatchLogRecordProcessor.java

```diff
@@ -45,8 +49,8 @@ public final class BatchLogRecordProcessor implements LogRecordProcessor {
       BatchLogRecordProcessor.class.getSimpleName() + "_WorkerThread";
   private static final AttributeKey<String> LOG_RECORD_PROCESSOR_TYPE_LABEL =
       AttributeKey.stringKey("logRecordProcessorType");
-  private static final AttributeKey<Boolean> LOG_RECORD_PROCESSOR_DROPPED_LABEL =
-      AttributeKey.booleanKey("dropped");
+  private static final AttributeKey<String> LOG_RECORD_PROCESS_STATUS_LABEL =
```

Comment:
Strongly recommend avoiding using the term "label" to refer to OpenTelemetry attributes. Let's try and be consistent in our naming. 🙏🏻